src/Microsoft.Azure.SignalR.Common/Constants.cs

```diff
@@ -22,7 +22,7 @@ internal static class Constants
     public const string AsrsDefaultScope = "https://signalr.azure.com/.default";
 
 
-    public const int DefaultCloseTimeoutMilliseconds = 30000;
+    public const int DefaultCloseTimeoutMilliseconds = 5000;
```

Comment:
This is claimed in the [GracefulShutdownOptions](https://github.com/Azure/azure-signalr/blob/dev/src/Microsoft.Azure.SignalR/GracefulShutdownOptions.cs#L21C67-L21C89), also need an update?